squashfs3: fix build with uClibc

The squashfs3 package uses the old get_nprocs() GNU extension which does not
exist in uClibc. This has already been fixed in newer squashfs releases
(>=4.0).

Signed-off-by: Markos Chandras <markos.chandras@imgtec.com>
Index: squashfs3-3.4/squashfs-tools/mksquashfs.c
===================================================================
--- squashfs3-3.4.orig/squashfs-tools/mksquashfs.c
+++ squashfs3-3.4/squashfs-tools/mksquashfs.c
@@ -3178,7 +3178,7 @@ void initialise_threads()
 			processors = 1;
 		}
 #else
-		processors = get_nprocs();
+		processors = sysconf(_SC_NPROCESSORS_CONF);
 #endif
 	}
 
Index: squashfs3-3.4/squashfs-tools/unsquashfs.c
===================================================================
--- squashfs3-3.4.orig/squashfs-tools/unsquashfs.c
+++ squashfs3-3.4/squashfs-tools/unsquashfs.c
@@ -2428,7 +2428,7 @@ void initialise_threads(int fragment_buf
 			processors = 1;
 		}
 #else
-		processors = get_nprocs();
+		processors = sysconf(_SC_NPROCESSORS_CONF);
 #endif
 	}
 
